governance/polls/Ratification Poll for Pairwyse Licensure (MIP66) - April 11, 2022.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Pairwyse Licensure (MIP66) - April 11, 2022.
+summary: The Pairwyse Foundation has implemented an out-of-box smart contract solution and deployed software products for provisioning fixed-rate DAI loans of arbitrary term-length via the Pairwyse protocol built atop Maker’s credit and deposit facilities.
+discussion_link: https://forum.makerdao.com/t/mip66-pairwyse-licensure/13554
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-25T16:00:00
+---
+# Ratification Poll for Pairwyse Licensure (MIP66) - April 11, 2022.
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, April 11 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP66 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Pairwyse is an active protocol and blueprint for creating bespoke fixed-rate DAI lending and investment Agreements on MakerDAO between borrowers and lenders. We wish to thank the Maker Community for its previous feedback concerning the negative impact of patents on open source projects and potential concentration risks posed by fixed-rate innovations currently incubating in the Maker ecosystem. In posting this MIP, we wish to directly address these shared concerns with a fresh proposal we feel is favorable for MakerDAO, its contributors, its users and all current and future DeFi innovators throughout the collective ecosystem.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/2e55a730e76594880d6f4b69104b0816e7b5720b)
```

Comment:
Use this link: https://github.com/makerdao/mips/blob/2e55a730e76594880d6f4b69104b0816e7b5720b/MIP66/MIP66.md